### PR TITLE
Zip store and deps together

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Addressing #52 

Also removing artifacts.xml from the runtime store zip.

This needs to be approved for rel/2.0.0-preview1

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
The zip structure is 
```
$(zipRoot)
\_ store
  \_ x64
    \_ ...
  \_ x86
    \_ ...
\_ additionalDeps
  \_ Microsoft.NETCore.App
    \_ 2.0.0
      \_ *.deps.json
```

**Comment by muratg** [comment]
cc @BillHiebert @barrytang

**Comment by Tratcher** [comment]
It was supposed to be:
```
\_ additionalDeps
  \_ [PackageName]
    \_ Microsoft.NETCore.App
      \_ 2.0.0
        \_ deps.json
```
@glennc

**Comment by Tratcher** [comment]
@BillHiebert e.g.
c:\program files\dotnet\additionalDeps\Microsoft.AspNetCore.ApplicationInsights.HostingStartup

**Comment by JunTaoLuo** [comment]
:up::date:

File: NuGet.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
```
Comment (pranavkm) [inline NuGet.config]: Huh?

File: build/common.props [hunk 2]
```diff
@@ -171,11 +171,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <RuntimeStorePackageReference Include="@(FullMetaPackagePackageReference)" />
-    <RuntimeStorePackageReference Include="Microsoft.AspNetCore" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
+    <RuntimeStorePackageReference Include="Microsoft.Extensions.Primitives" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
     <RuntimeStorePackageReference Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
-    <RuntimeStorePackageReference Include="Microsoft.AspNetCore.AzureAppServicesIntegration" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
```
Comment (pranavkm) [inline build/common.props]: Why are these missing?

File: build/repo.targets [hunk 3]
```diff
@@ -59,23 +59,29 @@
...
       </PackageStoreManifestFiles>
+      <PackageCacheFiles Include="$(PackageCacheOutputPath)**\*" Exclude="$(PackageCacheOutputPath)**\artifact.xml" />
       <DepsFiles Include="$(DepsOutputPath)**\*" />
     </ItemGroup>
 
-    <ZipArchive File="$(OutputZip)" SourceFiles="@(PackageCacheFiles)" WorkingDirectory="$(OutputPath)" />
-    <ZipArchive File="$(DepsOutputZip)" SourceFiles="@(DepsFiles)" WorkingDirectory="$(DepsOutputPath)" />
+    <Move SourceFiles="%(PackageStoreManifestFiles.FullPath)" DestinationFiles="$(ArtifactsDir)%(PackageStoreManifestFiles.DestinationFile)" />
+
+    <Copy SourceFiles="@(PackageCacheFiles)" DestinationFolder="$(ArtifactsZipDir)store/%(RecursiveDir)" />
```
Comment (pranavkm) [inline build/repo.targets]: back slashes

File: NuGet.config (hunk 1 above)
Comment (JunTaoLuo) [inline NuGet.config]: Gah, checked in my workarounds. https://github.com/aspnet/Common/pull/220

File: build/common.props (hunk 2 above)
Comment (JunTaoLuo) [inline build/common.props]: Oops this was for testing, need to revert

**Pull request merged**